WordPress/Classes/Services/ReaderPostService.m

```diff
@@ -431,6 +431,19 @@ - (void)flagPostsFromSite:(NSNumber *)siteID asBlocked:(BOOL)blocked
     }];
 }
 
+- (NSUInteger)numberOfReadItLaterPosts
+{
+    NSFetchRequest *request = [NSFetchRequest fetchRequestWithEntityName:@"ReaderPost"];
```

Comment:
You could do `NSStringFromClass([ReaderPost class])` so should we ever rename the class via refactoring this code will get updated for free.